docs/configuration.md

```diff
@@ -1481,6 +1481,17 @@ Apart from these, the following properties are also available, and may be useful
     How many batches the Spark Streaming UI and status APIs remember before garbage collecting.
   </td>
 </tr>
+<tr>
+  <td><code>spark.streaming.listeners</code></td>
```

Comment:
Why is this not `spark.streaming.extraListeners`, similar to `spark.extraListeners`?